index.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
           * Always delay requires, otherwise your plugin will cause trouble
           * with db-migrates performance and generates issues to your users.
           */
-        require('typescript').register();
+        var typescriptModule = require('typescript');
+        if (typescriptModule && typescriptModule.register) {
```

Comment:
Ok, but does this plugin then do even work currently?
If not may be switching over to the mentioned https://github.com/TypeStrong/ts-node for the time being might be an option?